types/modules/admob/AdMobComponent.d.ts

```diff
@@ -0,0 +1,54 @@
+/// <reference types="react" />
+import * as React from 'react';
+export interface AdMobComponentProps {
```

Comment:
This is missing the required `size: string` and `unitId: string` properties. It should also extend `ViewPropTypes` from `react-native`